source/index.html.md.erb

```diff
@@ -87,3 +87,12 @@ and, assuming the user is still authorized to access the resources in
 question.
 
 _Last updated: <%= Date.today.strftime("%-d %B %Y") %>_
+
+### Versioning
+
+When an API changes in a way that is backwards-incompatible, a new version number of the API will be published.
```

Comment:
Could we add:

- explicitly saying when we make non-breaking changes to the API this will not result in a version bump
- in the event fields or methods will be removed in the next version, deprecation information will be included in the API response (GDS [recommends](https://www.gov.uk/guidance/gds-api-technical-and-data-standards#set-clear-deprecation-policies) a "Warning" header for this)
- we will publish a changelog containing all breaking and non-breaking changes

Reply:
👍